src/FSCommon.h

```diff
@@ -23,13 +23,24 @@
 #endif
 
 #if defined(ARCH_ESP32)
+
+#if CONFIG_IDF_TARGET_ESP32S3
+// ESP32S3 version
+#include "FFat.h"
+#define FSCom FFat
+#define FSBegin() FSCom.begin(true)
+#define FILE_O_WRITE "w"
+#define FILE_O_READ  "r"
+#else
 // ESP32 version
```

Comment:
Again, i'd rather stick to littlefs for uniform layouts across supported platforms.